### PR TITLE
print_every_n python cv

### PR DESCRIPTION
**Pull request by aszalucha**
I added option print every n to python package. It's similiar to R's xgb.cv option print.every.n

### CODE REVIEW

File: python-package/xgboost/training.py [hunk 1]
```diff
@@ -282,6 +284,11 @@ def cv(params, dtrain, num_boost_round=10, nfold=3, metrics=(),
     results = []
     cvfolds = mknfold(dtrain, nfold, params, seed, metrics, fpreproc)
     for i in range(num_boost_round):
+        if print_every_n:
+            if i == 0 or i % print_every_n == 0:
```
Comment (sinhrks) [inline python-package/xgboost/training.py]: I think ``i == 0`` is not needed here.

File: python-package/xgboost/training.py [hunk 2]
```diff
@@ -282,6 +284,11 @@ def cv(params, dtrain, num_boost_round=10, nfold=3, metrics=(),
...
+                show_progress = True
+            else:
+                show_progress = False
```
Comment (sinhrks) [inline python-package/xgboost/training.py]: Personally feel  ``show_progress`` should be prioritized, for example what should happen ``show_progress=True`` and ``print_every_n=5``?

**Comment by aszalucha** [comment]
I think, that if somebody puts these 2 parameters, he wants to show progress every 5 iterations.

File: python-package/xgboost/training.py (hunk 1 above)
Comment (aszalucha) [inline python-package/xgboost/training.py]: Yeah, you're right.

**Comment by terrytangyuan** [comment]
@aszalucha Is this still active? 
See #621 for similar PR, closing this for now. Feel free to re-open it when ready.

**Pull request closed without merging**